### PR TITLE
Fix scrollbar issue

### PR DESCRIPTION
**Pull request by sjanzou**
Fixes widgets going off page bottom but resets to top of page. Cashflow also works for BOM and FOM configurations

### CODE REVIEW

**Review by brtietz — APPROVED** [review]
The fix looks great! Is there a reason to leave so much commented-out code? I generally find git history more helpful since the commit message provides good context on why things were changed or removed.

**Review by mjprilliman — COMMENTED** [review]
I'm still seeing some issues on the battery example with the bottom of the page cutting off. It seems that when you open one collapsible tab the page appropriately resizes, but upon opening a second tab the page size cuts off the bottom again and the top shifts down. 

I can note that in my testing for the ME pages programatically opening the collapsible tabs by manipulating the collapsible var used in the startup page does seem to automatically fix the page size each time. Not sure how helpful that is for the battery case but something I discovered.

**Review by cpaulgilman — APPROVED** [review]

File: src/inputpage.cpp [hunk 1]
```diff
@@ -138,8 +138,11 @@ ActiveInputPage::~ActiveInputPage()
 
 wxRect ActiveInputPage::ScaleRect( const wxRect &r )
 {
-	return wxRect( (int)(r.x*m_scaleX), (int)(r.y*m_scaleY),
-		(int)(r.width*m_scaleX), (int)(r.height*m_scaleY));
+	// testing
+//	return wxRect((int)(r.x * m_scaleX), (int)(r.y ),
+//		(int)(r.width * m_scaleX), (int)(r.height * m_scaleY));
```
Comment (dguittet) [inline src/inputpage.cpp:143]: please remove unused

File: src/inputpage.cpp [hunk 2]
```diff
@@ -318,6 +321,8 @@ void ActiveInputPage::OnPaint( wxPaintEvent & )
 			if (!objs[i]->IsVisible())
 			{
 				rct = ScaleRect(objs[i]->GetGeometry());
+				// testing
+				//if (i == 0) rct.y = 0;
```
Comment (dguittet) [inline src/inputpage.cpp:325]: please remove

**Review by dguittet — CHANGES_REQUESTED** [review]
Thanks for fixing it so quickly! Agree with @brtietz that unnecessary comments should be removed. If there is extra information you'd like to track, perhaps adding them as comments to this Issue / PR will be a good reference instead.

**Comment by sjanzou** [comment]
Changes requested made and scroll bar issue in #405 resolved.

**Comment by sjanzou** [comment]
Address all issues and closes #405

**Comment by sjanzou** [comment]
Closes #405

**Pull request merged**